ajax/libs/react-block-ui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-block-ui",
+  "filename": "reactblockui.min.js",
+  "description": "Block UI for react",
+  "homepage": "https://availity.github.io/react-block-ui/",
+  "keywords": [
+    "availity",
+    "loader",
+    "blockui",
+    "redux",
+    "react",
+    "component",
+    "components",
+    "react-component",
+    "ui"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Availity/react-block-ui.git"
+  },
+  "license": "MIT",
+  "author": "Evan Sharp <evan.sharp@availity.com> (https://github.com/TheSharpieOne)",
+  "npmName": "react-block-ui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "reactblockui.min.js*",
+        "style.css"
```

Comment:
@TheSharpieOne thank you for your contribution.
May I ask the the `reactblockui.min.js` is compressed from `reactblockui.js`?
If yes. CDNJS can host the `reactblockui.js*` file.

Would you want to add `reactblockui.js`?
If yes.
I think we can modify from A to B as the followings.

- A 
```js
    {
      "basePath": "dist",
      "files": [
        "reactblockui.min.js*",
        "style.css"
      ]
    }
```

- B
```js
    {
      "basePath": "dist",
      "files": [
        "**/+(reactblockui.!(cjs|es)|style)*+(css|js|map)"
      ]
    }
```

What do you think?
Thank you.